group_vars/sn06.yml

```
@@ -470,7 +470,7 @@ galaxy_dynamic_job_rules_src_dir: files/galaxy/dynamic_rules
 galaxy_dynamic_job_rules_dir: "{{ galaxy_root }}/dynamic_rules"
 galaxy_dynamic_job_rules:
   - usegalaxy/joint_destinations.yaml
-  - usegalaxy/sorting_hat.py
+  - usegalaxy/sorting_hat.yaml
```

```suggestion
  - usegalaxy/sorting_hat.py
```

`usegalaxy/sorting_hat.yaml` does not exist...